egs/tedlium3/ASR/transducer_stateless/asr_datamodule.py

```diff
@@ -209,10 +210,22 @@ def train_dataloaders(self, cuts_train: CutSet) -> DataLoader:
             logging.info(
                 f"Time warp factor: {self.args.spec_aug_time_warp_factor}"
             )
+            # Judge num_frame_masks for SpecAugment according to Lhotse'version
+            num_frame_masks = (
```

Comment:
I just feel this kind of usage of ternary expression in python is hard to follow.

How about
```python3
if xxx:
  num_frame_masks = 2
else:
  num_frame_masks = 10
```

Reply:
I think it's ok.

Reply:
If the change in this file is ok, and then I will modify other asr_datamodule.py files.

Reply:
> If the change in this file is ok, and then I will modify other asr_datamodule.py files.

Yes, it looks good to me.

Reply:
But I suggest you to create a new PR for it.